lib/Zonemaster/Backend/Config.pm

```diff
@@ -370,134 +348,67 @@ sub ListLanguageTags {
     return @langtags;
 }
 
-sub PollingInterval {
-    my ($self) = @_;
-
-    return $self->DB_polling_interval;
-}
-
-
-=head2 MaxZonemasterExecutionTime
-
-=head3 INPUT
 
-'max_zonemaster_execution_time' from [ZONEMASTER] section in ini file. See
-L<https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#max_zonemaster_execution_time>.
+=head2 DB_engine
```

Comment:
Can you move all these POD (DB_engine ... ZONEMASTER_age_reuse_previous_test) above line 266 with the other ones ? I think it will make more sense to have them all together (however I should say that I don't know much on POD, so if there is a reason to keep them here, I can live with that :) )

Reply:
POD doesn't care about order. There are coverage tools for verifying that every named subroutine has a corresponding POD heading but otherwise you may do as you please.

For this PR I've mainly tried to do something at least half-way reasonable regarding the placement of the POD while also keeping down the size of the diff. But I'll take another look.

Reply:
This was just to have the POD close to the code (if you prefer reading the code file directly). Thanks for the update

Reply:
I completely agree. The only reason I didn't make it so in the first place was a practical one. I had a large set of changes that I was trying to rebase into a series of PRs, so having smaller diffs made a big difference at the time. But I'm glad I got to set it straight.